pdns/iputils.cc

```diff
@@ -436,3 +436,36 @@ bool isTCPSocketUsable(int sock)
 
   return false;
 }
+/* mission in life: parse three cases
```

Comment:
yet I see 4 :)

Reply:
heh, that's a copy-paste form the original